lib/fileio/utils.go

```diff
@@ -21,8 +21,16 @@ import (
 	"io/ioutil"
 	"os"
 	"path/filepath"
+
+	"github.com/uber/makisu/lib/utils"
 )
 
+// fileOwners returns the uid & gid that own the file.
+func fileOwners(fi os.FileInfo) (uid int, gid int) {
```

Comment:
may be more intuitive to call it getFileOwners.

Reply:
ok